### PR TITLE
Zizombi

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

https://github.com/Darkrp-community/OpenKeep/pull/400 is this PR but fixed
the mobs are tested and work properly

## Why It's Good For The Game

more pve

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/npc/zizombies.dm [hunk 1]
```diff
@@ -0,0 +1,611 @@
...
+			beltl = /obj/item/clothing/neck/roguetown/psicross
+		if(2) //zizombie Warrior
+			r_hand = /obj/item/rogueweapon/halberd
+			pants = /obj/item/clothing/under/roguetown/grenzelpants
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/light
+			gloves = /obj/item/clothing/gloves/roguetown/grenzelgloves
+			belt = /obj/item/storage/belt/rogue/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/grenzelhoft
+			head = /obj/item/clothing/head/roguetown/helmet/sallet/visored
+			neck = /obj/item/clothing/neck/roguetown/bervor
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/zizombies.dm:564]: Correct path is /obj/item/clothing/neck/roguetown/bevor

File: code/modules/mob/living/carbon/human/npc/zizombies.dm [hunk 2]
```diff
@@ -0,0 +1,611 @@
...
+			beltl = /obj/item/clothing/neck/roguetown/psicross
+		if(3) //zizombie Warrior
+			r_hand = /obj/item/rogueweapon/eaglebeak
+			pants = /obj/item/clothing/under/roguetown/grenzelpants
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/light
+			gloves = /obj/item/clothing/gloves/roguetown/grenzelgloves
+			belt = /obj/item/storage/belt/rogue/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/grenzelhoft
+			head = /obj/item/clothing/head/roguetown/helmet/sallet/visored
+			neck = /obj/item/clothing/neck/roguetown/bervor
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/zizombies.dm:577]: Ditto

File: code/modules/mob/living/carbon/human/npc/zizombies.dm [hunk 3]
```diff
@@ -0,0 +1,611 @@
...
+		if(4) //zizombie Warrior
+			r_hand = /obj/item/rogueweapon/sword/iron/short
+			l_hand = /obj/item/rogueweapon/huntingknife/idagger
+			pants = /obj/item/clothing/under/roguetown/grenzelpants
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/light
+			gloves = /obj/item/clothing/gloves/roguetown/grenzelgloves
+			belt = /obj/item/storage/belt/rogue/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/grenzelhoft
+			head = /obj/item/clothing/head/roguetown/helmet/sallet/visored
+			neck = /obj/item/clothing/neck/roguetown/bervor
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/zizombies.dm:591]: Ditto

File: code/modules/mob/living/carbon/human/npc/zizombies.dm [hunk 4]
```diff
@@ -0,0 +1,611 @@
...
+			beltl = /obj/item/clothing/neck/roguetown/psicross
+		if(5) //zizombie Warrior
+			r_hand = /obj/item/rogueweapon/battle
+			pants = /obj/item/clothing/under/roguetown/grenzelpants
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/light
+			gloves = /obj/item/clothing/gloves/roguetown/grenzelgloves
+			belt = /obj/item/storage/belt/rogue/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/grenzelhoft
+			head = /obj/item/clothing/head/roguetown/helmet/heavy/knight
+			neck = /obj/item/clothing/neck/roguetown/bervor
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/zizombies.dm:604]: Ditto

File: code/modules/mob/living/carbon/human/npc/zizombies.dm [hunk 5]
```diff
@@ -0,0 +1,611 @@
...
+	var/loadout = rand(1,5)
+	switch(loadout)
+		if(1) //zizombie Warrior
+			r_hand = /obj/item/rogueweapon/greatsword/zwei
+			pants = /obj/item/clothing/under/roguetown/grenzelpants
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/light
+			gloves = /obj/item/clothing/gloves/roguetown/grenzelgloves
+			belt = /obj/item/storage/belt/rogue/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/grenzelhoft
+			head = /obj/item/clothing/head/roguetown/helmet/heavy/pigface
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/zizombies.dm:550]: Correct path is /obj/item/clothing/head/roguetown/helmet/heavy/hounskull

Comment (Dorodomki) [reply code/modules/mob/living/carbon/human/npc/zizombies.dm]: thanks for the help broski

**Comment by ThePainkiller** [comment]
1. No psycrosses (they should be burning from the inside out being zizombies wearing the symbols of the forgotten one)
2. **No bevors**
3. No way in hell Jose **no Hounskulls, ever, at all. Sallets at most. No free endgame tier armor for people, let alone for mobs that should spawn as ambushes or for siege events.**

**Comment by Dorodomki** [comment]
> 1. No psycrosses (they should be burning from the inside out being zizombies wearing the symbols of the forgotten one)
> 2. **No bevors**
> 3. No way in hell Jose **no Hounskulls, ever, at all. Sallets at most. No free endgame tier armor for people, let alone for mobs that should spawn as ambushes or for siege events.**

the grenzelhoft mercs are the top PVE minibosses, they won't spawn normally and already warned about their danger with the eventmins, these guys are supposed to spawn in very small teams

**Comment by ThePainkiller** [comment]
> the grenzelhoft mercs are the top PVE minibosses, they won't spawn normally and already warned about their danger with the eventmins, these guys are supposed to spawn in very small teams

Not even our grenzel mercs spawn with hounskull-adjacent equipment
No danger justifies just loot piñata, not even if they're ominous forces and whatnot, if anything it's better for admins to outright gear one particular mob and hand it particular equipment

These mobs with endgame tier equipment, specially if player controlled, will be as impossible to deal with as a vampire due to their `NOROGSTAM` and crit protection, plus no apparent crit weakness nor silver/fire weakness. Don't overtune mobs just because "muh elite endgame", no town wants to be steamrolled on wave events specially after everyone is hungry, thirsty, sleepy, tired from the fight, stressed, and facing elite mobs with infinite stamina

Plus, again
No handing endgame gear that is only obtainable through smithing
There's a reason I made the smithing quality PR port and is to avoid people ease of access to these things, don't invalidate blacksmiths AGAIN

**Review by ThePainkiller — APPROVED** [review]
Hopefully these will be balanced enough not to be an issue, we'll see after player feedback.

**Pull request merged**